EXAMPLES/MATRIX_MARKET/arpackSolver.hpp

```diff
@@ -670,6 +672,21 @@ class arpackSolver {
       }
     };
 
+    int initPointerSize(a_int & iparamSz, a_int & ipntrSz, string const & aeupd) {
+      iparamSz = 11; ipntrSz = 14;
+      if (aeupd == "aupd") {
+        if (is_same<RC, double>::value && symPb) ipntrSz = 11;
+        if (is_same<RC,  float>::value && symPb) ipntrSz = 11;
+        return 0;
+      }
+      else if (aeupd == "eupd") {
```

Comment:
```suggestion
        return 0;
      }
      if (aeupd == "eupd") {
```

The else is useless as we have a return just above

Reply:
Sure

Reply:
fyi, you could just have press the "commit suggestion" button ;)

Reply:
OK, now I know!